backend/app/main.py

```
@@ -27,14 +27,34 @@
+    log_handler = logging.FileHandler(settings.LOG_FILE)
+    formatter = jsonlogger.JsonFormatter(
+        fmt="%(asctime)s %(levelname)s %(name)s %(message)s %(request_id)s",
+        json_ensure_ascii=False
+    )
+    if log_handler and formatter:
+        log_handler.setFormatter(formatter)
+        logger.handlers = [log_handler]
+        logger.setLevel(getattr(logging, settings.LOG_LEVEL, logging.INFO))
+        # Verify logger.error is callable
+        if not callable(getattr(logger, 'error', None)):
+            raise ValueError("Logger error method is not callable")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)